src/eos-autoupdater.c

```diff
@@ -61,6 +63,7 @@ static const char *UPDATE_STAMP_DIR = LOCALSTATEDIR "/eos-updater";
 static const char *UPDATE_STAMP_NAME = "eos-updater-stamp";
 
 static const char *CONFIG_FILE_PATH = SYSCONFDIR "/eos-updater.conf";
+static const char *STATIC_CONFIG_FILE_PATH = DATADIR "/" PACKAGE_NAME "/etc/eos-updater.conf";
```

Comment:
Oh, and if you do want a separate subdir, I'd probably call it `conf` or `config` rather than carrying forward the `etc` misnomer.